### PR TITLE
Fix yaml loading

### PR DESCRIPTION
**Pull request by tycooon**
Addresses #132.

### CODE REVIEW

File: lib/qonfig/plugins/vault/loaders/vault.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@ class Qonfig::Loaders::Vault < Qonfig::Loaders::Basic
   # @api private
   # @since 0.25.0
   VAULT_EXPR_EVAL_SCOPE = BasicObject.new.__binding__.tap do |binding|
-    Object.new.method(:freeze).unbind.bind(binding.receiver).call
+    Object.new.method(:freeze).unbind.bind_call(binding.receiver)
```
Comment (0exp) [inline lib/qonfig/plugins/vault/loaders/vault.rb:11]: need to add `@version` tag :)

File: lib/qonfig/loaders/yaml.rb [hunk 2]
```diff
@@ -12,7 +12,7 @@ class << self
     # @api private
     # @since 0.2.0
     def load(data)
-      ::YAML.load(ERB.new(data).result)
+      ::YAML.safe_load(ERB.new(data).result, permitted_classes: [Symbol], aliases: true)
```
[the remark this answers is not in the document]
Comment (0exp) [reply lib/qonfig/loaders/yaml.rb:15]: need to add `@version` tag :)

File: qonfig.gemspec [hunk 3]
```diff
@@ -6,7 +6,7 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'qonfig/version'
```
Comment (0exp) [inline qonfig.gemspec:6]: i think, we can bump the current version 🤔

File: lib/qonfig/loaders/yaml.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (0exp) [reply lib/qonfig/loaders/yaml.rb:16]: I want to use `load` in order to support all variants of yaml loading AND the safety should be controlled on the developer side. Instead: we can add the configurable variant of this logic, and add an option to the Configuration layer 🤔 What do you think? (by this way i agree that we should youse safe_load and optionally support the `unsafe_load` by configs or method options)

[the remark this answers is not in the document]
Comment (tycooon) [reply lib/qonfig/loaders/yaml.rb:16]: We can use `unsafe_load` here for now, in order to keep the old behavior. Adding configuration sounds like a new feature for me :)

[the remark this answers is not in the document]
Comment (0exp) [reply lib/qonfig/loaders/yaml.rb]: :pray: let's use `load`) (`unsafe_load` was added in new Ruby/Psych releases)
and i will add new feature to the roadmap (I agree that `save_load` is better for everyone)

**Pull request merged**